app/Cargo.toml

```diff
@@ -6,9 +6,11 @@ publish = false
 edition = "2021"
 
 [dependencies]
-tokio = { version = "1.0", features = ["full"] }
-tonic = "0.7.2"
-prost = "0.10.4"
+tokio = { version = "1.0", features = ["full"], optional = true }
```

Comment:
Is it possible to specify tokio features needed instead of `full`?

Reply:
Done.